data/mods/TropiCataclysm/monsters/tropical_nether.json

```
@@ -390,7 +390,7 @@
   {
     "id": "mon_fish_flying_tiny",
     "type": "MONSTER",
-    "name": { "str_sp": "tiny flying fish" },
+    "name": { "str": "tiny flying fish", "str_pl": "tiny flying fishes" },
```

I think `fishes` is only used to refering to multiple species, and the plural for multiple individuals of the same species is simply `fish`.

---

So you think that all of them should simply use `str` without `str_pl`?

---

You can use `str_sp` to make the plural the same as the singular, like this:

```suggestion
    "name": { "str_sp": "tiny flying fish" },
```